changes-set.txt

```diff
@@ -15,11 +15,87 @@ The ones WITH "Notes" may have to be processed manually.
 PROPOSED:
 Date      Old       New         Notes
 proposed  syl       imtri       (analogous to *bitr*, sstri, etc.)
-proposed  sylib     imbitri     etc.
-proposed  sylbid    biimtrd     etc.
-proposed  sylbird   biimtrrd    etc.
-proposed  syl5*     *trid       (syl5bi -> biimtrid; syl5eqel -> eqeltrid;etc.)
-proposed  syl6*     *trdi
+                                there is less agreemnt on renaming syl
+                                than others here
+proposed  syld      imtrd
+proposed  syldc     imtrdcom
+proposed  syldd     imtrdd
+proposed  sylbi     biimtri
+proposed  sylib     imbitri
+proposed  sylbb     bitriim
+proposed  sylibr    imbitrri
+proposed  sylbir    biimtrri
+proposed  sylbbr    bitrriim
+proposed  sylbb1    bitr3iim
+proposed  sylbb2    bitr4iim
+proposed  sylibd    imbitrd
+proposed  sylbid    biimtrd
+proposed  sylibrd   imbitrrd
+proposed  sylbird   biimtrrd
+proposed  syl5com   imtridcom
+proposed  syl5      imtrid      alternate proposal: sylid
+proposed  syl56     imtridi
+proposed  syl5d     imtridd
```

Comment:
isn't this inconsistent with the interpretation of i and d representing hypotheses?

Reply:
`sylid` and `syldi` aren't as systematic as `imtrid` and `imtrdi`, agreed.

That was my thinking in putting them down as alternative proposals rather than saying this one is settled.

Reply:
Agreed, `syl5d` doesn't fit any of the patterns especially well. One question here is whether we are determined to get rid of the numbers (or at least the numbers `5` and `6`), which is mostly pretty doable if we adopt the `id` and `di` conventions, but does leave a few which are harder to name.

Reply:
Hmm. Maybe `d` is an operation that adds a deduction to all hypotheses.

Then for a specific hypothesis adding an antecedent would be `1` or `2`, i.e. the index of the hypothesis.
```
syl5    syl2
syl6    syl1
syl56   syl13 (hmm)
syl5d   syl2d
syl6d   syl1d

syl11   sylcom1 (syl5 with commuted hypothesis)
```
This is somewhat unsatisfying, as syl5 feels "earlier" than syl6 because in the deductive hypothesis, `ph -> ps -> ch`, syl5 replaces ps while syl6 replaces ch

Reply:
It is an interesting thought process, but like the one which led me to wonder about `3syl` to `imtriii` as described at https://github.com/metamath/set.mm/pull/3556#issuecomment-1752200990 , I'm not sure it quite falls into place.

I think we can start with the easier cases and realize there are going to be some things which don't fit as readily into a systematic scheme. We know that our names will never be completely systematic (I mean, given our other goals like wanting them short and memorable too).